Sprint-1/exercises/count.js

```diff
@@ -4,3 +4,6 @@ count = count + 1;
 
 // Line 1 is a variable declaration, creating the count variable with an initial value of 0
 // Describe what line 3 is doing, in particular focus on what = is doing
+
+
+// Line 3 is reassigning the value of the variable "count", to the value of "count + 1"
```

Comment:
I was referring to the term "increment". We can describe `count = count +1` as "increment the value of count by 1".